### PR TITLE
Add pytorch check

### PR DESCRIPTION
**Pull request by jgphpc**

### CODE REVIEW

**Comment by vkarak** [comment]
Is there a reason why this is a wip?

File: cscs-checks/apps/tensorflow/keras_check.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
+# Copyright 2016-2020 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# ReFrame Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+imagenet_tar = '/apps/common/UES/reframe/resources/training/imagenet.tar'
```
Comment (henrique) [inline cscs-checks/apps/tensorflow/keras_check.py:10]: this is only about 10% of the actual imagenet dataset
(I don't know if it's necessary to add it to reframe, though)
Can we keep this PR just about PyTorch, please?

**Comment by vkarak** [comment]
@jgphpc Is this still a wip?

**Comment by jgphpc** [comment]
moved to https://github.com/eth-cscs/reframe/pull/1675

**Pull request closed without merging**